### PR TITLE
Add new helpful messages in `developer-tools` to validation docs

### PR DESCRIPTION
**Pull request by ivarpruijn**

### CODE REVIEW

**Review by sdirosa — APPROVED** [review]
Nice addition! One small comment, minor.

**Review by jansenbob — APPROVED** [review]
Looking good. Two optional tweaks to the copy

File: docs/docs/tracking/core-concepts/validation.md [hunk 1]
```diff
@@ -5,29 +5,87 @@ title: Validation
...
+```js
+if (process.env.NODE_ENV.startsWith('dev')) {
+  require('@objectiv/developer-tools');
+}
+```
+
+When loaded, it will self-inject globally, and logging & validation will be enabled.
+
+See two examples below. More validation rules are available, such as Location Contexts that are in the wrong 
+position in the stack, and will be expanded, e.g. redundant Contexts, or duplicate Contexts.
```
[the remark this answers is not in the document]
Comment (ivarpruijn) [reply docs/docs/tracking/core-concepts/validation.md:53]: Agree it's not very clear, will update

File: docs/docs/tracking/core-concepts/validation.md [hunk 2]
```diff
@@ -5,29 +5,87 @@ title: Validation
...
+in the browser console. Warnings are thrown for unknown tagging/tracking calls, missing properties, or 
+wrongly typed properties.
 
-* **Errors**: Warnings are thrown for unknown tagging/tracking calls, missing properties, or wrongly typed 
-  properties.
-* **Uniqueness**: When Developer Tools are enabled, if an Event `id` and its [Location Stack](locations.md) are not unique, an Error is thrown.
-* **Taxonomy**: When Developer Tools are enabled, trackers will perform the same validation the [Collector](/tracking/collector/introduction.md) does, client-side. This will help to identify issues early on, during development, such as missing Contexts or malformed Location Stacks.
+### Developer Tools
+If you also install the [`developer-tools`](https://www.npmjs.com/package/@objectiv/developer-tools) package,
+you get added validation for:
```
[the remark this answers is not in the document]
Comment (ivarpruijn) [reply docs/docs/tracking/core-concepts/validation.md:30]: Updated, thanks

**Comment by ivarpruijn** [comment]
LMK if you have any more comments @jansenbob @sdirosa, if not I'll merge this

**Comment by sdirosa** [comment]
> LMK if you have any more comments

Looks good to me 🚢

**Review by sdirosa — APPROVED** [review]

**Review by jansenbob — APPROVED** [review]
Ship!

**Pull request merged**